discovery-infra/tests/base_test.py

```diff
@@ -180,6 +181,20 @@ def _collect_virsh_logs(self, nodes, log_dir_name):
         infra_utils.run_command(f"journalctl --since \"{nodes.setup_time}\" "
                                 f"-u libvirtd -D /run/log/journal >> {libvird_log_path}", shell=True)
 
+    def _collect_journalctl(self, nodes, log_dir_name):
+        logging.info('Collecting journalctl\n')
+        os.makedirs(log_dir_name, exist_ok=True)
+        journal_ctl_path = os.path.join(log_dir_name, "journalctl")
+        os.makedirs(journal_ctl_path, exist_ok=False)
+        for node in nodes:
+            try:
+                journal = node.run_command(f'sudo journalctl')
+                journal_path = os.path.join(journal_ctl_path, node.name)
+                with open(journal_path, 'w') as _file:
+                    _file.write(journal)
+            except:
```

Comment:
Please be explicit. I guess in the case of `AttributeError` or `IndexError` you would want to crash in order to fix that failure.
Did you mean `paramiko.SSHException`?

Reply:
In the case the node is inactive a generic exception is raised:
https://github.com/openshift/assisted-test-infra/blob/master/discovery-infra/test_infra/controllers/node_controllers/node.py#L60

Also in the case where the node does not respond:
https://github.com/openshift/assisted-test-infra/blob/master/discovery-infra/test_infra/controllers/node_controllers/ssh.py#L55

Reply:
So we should change these `Exception`s as well to something more relevant. Like `RuntimeError` and `TimeoutError`